chromium_src/chrome/browser/themes/theme_helper.h

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_CHROME_BROWSER_THEMES_THEME_HELPER_H_
+#define BRAVE_CHROMIUM_SRC_CHROME_BROWSER_THEMES_THEME_HELPER_H_
+
+#define BRAVE_THEME_HELPER_H_    \
+ private:                        \
+  friend class BraveThemeHelper; \
+                                 \
+ public:
```

Comment:
nit: did you add `public:` intentionally?

Reply:
I thought we treated the BRAVE_FILENAME_H_ #defines as a template of sorts to add everything in one place. I suppose it doesn't need to be preemptively structured and then extended as needed. I can definitely just turn this to a single line friend statement since friend, as @iefremov noted below, isn't affected by access modifiers.

Reply:
no, this is correct. I think the docs may be wrong though because technically you don't need the `public:` at the end since this is placed right before `private`, but it's totally fine there